README.md

```diff
@@ -1,6 +1,6 @@
-# An Explicit Root Locale
+# Stable Formatting
 
-A TC-39 proposal to define the behaviour and properties of the `"und"` root locale.
+A TC-39 proposal to provide well-defined and powerful formatting methods.
```

Comment:
what does `powerful` mean here? What value it adds?

Reply:
How about:

> A TC39 proposal to bring stable Intl-inspired formatting options to ECMAScript

Reply:
That works.

```suggestion
A TC-39 proposal to bring stable Intl-inspired formatting options to ECMAScript.
```